agent/src/agent/pipeline/pipeline.py

```diff
@@ -448,3 +446,27 @@ class PipelineRetries(Entity):
     def __init__(self, pipeline_: Pipeline):
         self.pipeline_id = pipeline_.name
         self.number_of_error_statuses = 0
+
+
+class PipelineMetric(object):
```

Comment:
there's no need to inherit object, class definitions with and without object are the same